healthcare/api-client/v1/consent/README.rst

```diff
@@ -41,81 +42,77 @@ Install Dependencies
 #. Clone python-docs-samples and change directory to the sample directory you want to use.
 
     .. code-block:: bash
-
         $ git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git
```

Comment:
The code doesn't seem to show anymore, seems that the newlines are needed.

Reply:
Fixed.